DOCS.md

```diff
@@ -1,13 +1,21 @@
-Use this plugin for publishing files and artifacts to Chef Supermarket. You
-can override the default configuration with the following parameters:
+Global Parameters
+=================
+The following are global parameters used for configuration this plugin:
+* **user** - connects as this user
+* **private_key** - connects with this private key
+* **server** - (default: `'https://supermarket.chef.io'`) Supermarket server to connect to
+* **ssl_verify** - (default: `true`) Enable/Disable SSL verify
 
-* `` -
-
-## Example
-
-The following is a sample configuration in your .drone.yml file:
+Example
+=======
 
+### Minimal Definition
+This will upload the cookbook to a supermarket server
 ```yaml
 deploy:
-  chef_supermarket:
+  chef:
```

Comment:
just use `chef_supermarket` and it will be replaced with `plugins/drone-chef-supermarket`